packages/headless/src/controllers/index.ts

```diff
@@ -68,8 +68,6 @@ export type {
   FacetSearch,
   FacetSearchState,
   SpecificFacetSearchResult,
-  CoreFacet,
```

Comment:
Since you are removing `Core*` types from here, I guess they should be exported through the `src/index.ts` file otherwise they end up removed from the headless bundle.